src/sentry/tasks/relay.py

```diff
@@ -24,7 +24,7 @@
     time_limit=10,  # Extra 5 seconds to remove the debounce key.
     expires=30,  # Relay stops waiting for this anyway.
 )
-def build_project_config(public_key=None, **kwargs):
+def build_project_config(version: int = 3, public_key=None, **kwargs):
```

Comment:
This is where the issue was.

Reply:
I have no idea. I'm thinking how we could catch this with a test effectively...

Reply:
At this point I don't even understand why the original error happened.

Reply:
After syncing with @jan-auer, tl;dr it's due to the roll-out. See https://develop.sentry.dev/services/queue/